core/locals.tf

```
@@ -1,5 +1,6 @@
 locals {
-  app_name  = "devops-ic"
-  region    = "ap-southeast-1"
-  namespace = "${local.app_name}-${var.environment}"
+  app_name                      = "devops-ic"
+  region                        = "ap-southeast-1"
+  namespace                     = "${local.app_name}-${var.environment}"
+  s3_objects_expiration_in_days = 60
```

we can separate variables by module with new line, for better readability
```suggestion
  
  s3_objects_expiration_in_days = 60
```

---

Update to move it to s3 module's local in [4e7422c](https://github.com/nvminhtue/tyson_devops_ic/pull/28/commits/4e7422cdcf5fdc718c005c75fc0e2d5d5616d2ba) and [023e8f7](https://github.com/nvminhtue/tyson_devops_ic/pull/28/commits/023e8f79f1e7b700f98e64607c1a2a42274c802c)